popup.html

```diff
@@ -7,6 +7,10 @@
     <p id="header"></p>
     <canvas id="scroll-summary"></canvas>
     <ul id="stats"></ul>
+    <button id = "pause-current-site">Pause on this website</button>
+    <button id = "resume-current-site">Resume on this website</button>
+    <button id = "pause-all-sites">Pause on all websites</button>
+    <button id = "resume-all-sites">Resume on all websites</button>
```

Comment:
Suggesting: this looks quite clustered, can we have only two buttons for pause + unpause
![image](https://github.com/tin0312/walk-more-than-you-scroll/assets/51073515/d7de745c-2002-4eb2-9936-5c478f503c85)
After the tracking of a website is paused, the options become:
![image](https://github.com/tin0312/walk-more-than-you-scroll/assets/51073515/7db6c94d-489a-4e5f-bac0-f41694078716)